### PR TITLE
Update docs for distributed tracing insights component

### PR DESCRIPTION
**Pull request by jvanpeltnr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* Update documentation for the distributed tracing insights component
* A new version of this component is being released which streamlines the user flow

### CODE REVIEW

**Comment by jvanpeltnr** [comment]
re: the build fail, I added back in two old images that are still referenced by the translated versions of one of the pages changed. Should I assume that the old images will be removed when a PR is later created for the translations of the new content?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx [hunk 1]
```diff
@@ -90,4 +90,19 @@ For your OpenTelemetry data to appear in the **Summary** page, make sure it has
   </tbody>
 </table>
 
+### Distributed tracing insights [#distributed-tracing-insights]
+
+<img
```
Comment (akristen) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:95]: This is probably causing your build issues! Looks like this is being called, but was never imported into our images folder. If you add it in, the build should work.

File: src/i18n/content/jp/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ freshnessValidatedDate: '2023-07-19T00:00:00.000Z'
 translationType: machine
 ---
 
-import signalInsightHorizontalView from 'images/distributed-tracing_entity-signal-insight-horizontal.webp'
+import signalInsightHorizontalView from 'images/apm_screenshot-crop_distributed-tracing-in-the-apm-summary-page.webp'
```
Comment (akristen) [inline src/i18n/content/jp/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:12]: These get updated automatically, so we don't need to edit i18n files. I'll undo these in your PR

**Comment by akristen** [comment]
@jvanpeltnr I'm unclear about what changed re i18n images. I'll try to fix this as much as I can~ 

* If you want to update a screenshot, keep the same file name but change the image itself. So if you want to update image A named 'apmSummary.webp' with new image B, then you'd just name image B 'apmSummary.webp', add to the image folder + confirm you want to replace.
*  As for deleting images, we don't recommend deleting an image. Just remove the import statement and the embed, but leave the image in /images/. AFAIK our eng team automates removing any unused images.
* If you add images for an /en/ doc, we have a daily automated process that will carry that change over to our /jp/ and /kr/ docs.

File: src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx [hunk 3]
```diff
@@ -263,11 +263,12 @@ For more about how log patterns work, see [Log patterns](/docs/logs/ui-data/find
...
 
-For this example, let's say you want to dig deeper into the slow response times you saw in **Web transactions**. Click that section and you will see a list of traces that either go through your entire service or touch other areas of your ecosystem. New Relic will highligh areas where these other entities are _also_ having increases in response times. 
+The **Distributed tracing insights** chart shows you upstream and downstream entities that might cause increases in response time, error rate, or throughput of your service. 
 
-You can learn more in our [distributed tracing docs](/docs/distributed-tracing/concepts/introduction-distributed-tracing/).
+For example, let's say you want to investigate an increase in response time for a service, but the spike relates to the response time of an external call. If distributed tracing recorded a downstream entity that caused latency for your service around that time, you can view the entity and its change in performance in the list. Click the **View trace** button to see a distributed trace that shows performance changes.
```
Comment (akristen) [inline src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx:269]:
```suggestion
For example, let's say you want to investigate an increase in response time for a service, but the spike relates to an external call. If distributed tracing recorded a downstream entity that caused latency for your service, you can view that change in performance in this list. Click the **View trace** button to see a distributed trace that shows performance changes. 
```

Comment (jvanpeltnr) [reply src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx]: change pushed up

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx (hunk 1 above)
Comment (jvanpeltnr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:97]: fixed

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 4]
```diff
@@ -9,76 +9,49 @@ redirects:
...
-  src={signalSummariesWithAllClear}
-/>
-
-When you click on a signal card, a list of entities is displayed with significant changes in performance impact for that signal.
-If an entity meets this condition for multiple signals it will be listed under each of the respective cards.
-
 ## List of entities with a significant performance impact and change
 
+When any related trace entities had a significant change in their performance impact, you're shown a list of those performance signals and their change.
+If an entity is causing a significant change in performance impact, each of those signals will be shown in its own card in the list.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:36]:
```suggestion
* View impacted signals at a glance: When related trace entities undergo significant performance impact changes, you'll immediately see a list of the affected signals, along with their specific changes.
* Isolate key signals for quick action: If an entity is causing major performance impact shifts, each of those signals will be highlighted in its own dedicated card within the list, enabling you to quickly pinpoint and address potential issues.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 5]
```diff
@@ -9,76 +9,49 @@ redirects:
...
-<img
-  title="Signal summaries example"
-  alt="Example of signal summaries view"
-  src={signalSummariesWithAllClear}
-/>
-
-When you click on a signal card, a list of entities is displayed with significant changes in performance impact for that signal.
-If an entity meets this condition for multiple signals it will be listed under each of the respective cards.
-
 ## List of entities with a significant performance impact and change
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:33]:
```suggestion
## List of entities with a significant performance impact and change [#entity-list]
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 6]
```diff
@@ -9,76 +9,49 @@ redirects:
...
-
-Lists have an initial limit of 20 entities; in the event that there are more than 20 entities with a significant performance impact a **Show More** button appears at the bottom of the list.
+* The performance of the signal with a significant change in impact on your service, in aggregate and over time.
+* The name of the the entity.
+* The performance of the entity:
+    * **Count**: The number of traced calls to or from the listed entity, when it's in a call path with your entity.
+    * **Error rate**: The percent of calls to the listed entity that returned errors when called.
+    * **Exclusive time**: The average exclusive duration of calls that include your entity and the listed entity in the call path.
+* View Trace button: Click this button to view [trace details UI](/docs/distributed-tracing/ui-data/trace-details/) for a distributed trace where this performance impact between the other entity and your service was recorded.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:54]:
```suggestion
* **View Trace** button: Click this button to view [trace details UI](/docs/distributed-tracing/ui-data/trace-details/) for a distributed trace where this performance impact between the other entity and your service was recorded.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 7]
```diff
@@ -9,76 +9,49 @@ redirects:
...
 
 ## Compare with options
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:56]:
```suggestion
## Compare with options [#time picker]
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 8]
```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
...
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalInsightsWithControls}
+/>
+
+## Full-page View
+This view shows the same list of significant signals, with more real estate to scroll through longer lists.
+It also includes a filter option to focus on signals of a particular type.
+
+<img
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:84]:
```suggestion
## Full-page view [#full-page-view]

The full-page view shows the same list of significant signals but with more real estate to scroll through longer lists.
It also includes a filter option to focus on signals of a particular type.

<img
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 9]
```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
...
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalInsightsFullPage}
 />
 
+## No significant changes presented
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:90]:
```suggestion
## All-clear view [#all-clear-view]
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 10]
```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
...
+If no upstream or downstream services had a significant changes in their performance impact, the all clear view is shown.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:91]:
```suggestion

The all-clear view means no upstream or downstream services had a significant changes in their performance impact.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 11]
```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
...
+<img
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalSummariesWithAllClear}
+/>
+
+## When no tracing data is present
+You will see one of the following messages:
+* **No data for this time range** When viewing an older time window for your service that's beyond the retention period of your trace data. Select a time within the last week to see distributed tracing insights.
+* **No data from your entity** If your service is not reporting distributed tracing data. Follow links to our docs or the Guided Install to setup distributed tracing.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:102]:
```suggestion
## Understanding Missing Tracing Data Messages [#no-trace-data]

To ensure you're working with the most relevant distributed tracing insights, here's a breakdown of the messages you might encounter when data is unavailable, along with recommended actions:

* `No data for this time range`: 
     * This message indicates that you've selected a time window for your service that falls outside the current retention period for trace data.
     * To access insights, adjust the time range to focus on a period within the last week.
* `No data from your entity`:
     * This message suggests that your service isn't currently reporting distributed tracing data.
     * To unlock these valuable insights, set up distributed tracing for your service by following the [guided install](https://onenr.io/0OQM6lV3DwG). If you're new to distributed tracing, start by reading the [Distributed tracing: Planning guide](https://docs.newrelic.com/docs/distributed-tracing/concepts/distributed-tracing-planning-guide/).
```

Comment (ally-sassman) [reply src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:102]: @jvanpeltnr  How do they enable distributed tracing on the dependencies of their service? Is it the same process as enabling DT for the service?

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 12]
```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
...
+* **No data from related entities** If your service _is_ reported distributed tracing data, but no related entities are. Enable distributed tracing on the dependencies of your service to see insights.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:103]:
```suggestion
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx [hunk 13]
```diff
@@ -90,4 +92,19 @@ For your OpenTelemetry data to appear in the **Summary** page, make sure it has
...
+    title="distributed tracing in APM"
+    alt="A screenshot depicting distributed tracing information on the APM summary page"
+    src={apmDistributedTracingintheApmSummaryPage}
+/>
+You can use the **Distributed tracing insights** chart to learn whether entities downstream or upstream of your service might be causing increases in the response time, error rate, or throughput of your service.·
+
+For example, let's say you want to dig into a spike in the response time of your service when it's related to the external call response time.
+If distributed tracing recorded a downstream entity causing significantly increased latency for your service around that time, it will show that entity and its change in performance in the list.
+You can click the "View trace" button to see an example distributed trace where this performance behavior was recorded.
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:106]:
```suggestion
Click the **View trace** button to see an example distributed trace where this performance behavior was recorded.
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx [hunk 14]
```diff
@@ -90,4 +92,19 @@ For your OpenTelemetry data to appear in the **Summary** page, make sure it has
...
+
+You can learn more in our [distributed tracing insights docs](/docs/distributed-tracing/ui-data/related-trace-entity-signals/).
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:108]:
```suggestion
To learn more about distributed tracing, see our [distributed tracing insights docs](/docs/distributed-tracing/ui-data/related-trace-entity-signals/).
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx [hunk 15]
```diff
@@ -90,4 +92,19 @@ For your OpenTelemetry data to appear in the **Summary** page, make sure it has
...
+
 To learn about OpenTelemetry in other UI pages, see the [UI overview](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-view-your-data).
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:110]:
```suggestion
To learn about OpenTelemetry in other UI pages, see [Overview of OpenTelemetry data in the UI](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-view-your-data).
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx [hunk 16]
```diff
@@ -8,6 +8,8 @@ metaDescription: Tips for understanding the OpenTelemetry service Summary page i
 freshnessValidatedDate: never
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx:8]:
```suggestion
freshnessValidatedDate: 2024-01-08
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 17]
```diff
@@ -9,76 +9,49 @@ redirects:
 freshnessValidatedDate: 2023-07-19
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:9]:
```suggestion
freshnessValidatedDate: 2024-01-08
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx (hunk 11 above)
Comment (jvanpeltnr) [reply src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx]: Yes, same general process. We do have a page where you can see the config status for all apps you have access to, but you still have to know which entities are the dependencies for yours to use it, so the docs are just as useful in that case.

Comment (ally-sassman) [reply src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx]: @jvanpeltnr How does my suggestion look ^

Comment (jvanpeltnr) [reply src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx]: looks good - thanks!

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 18]
```diff
@@ -90,11 +63,42 @@ You may override the default compare with behavior by changing the value of the
...
+<img
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalSummariesWithAllClear}
+/>
+
+## When no tracing data is present
+You will see one of the following messages:
+* **No data for this time range** When viewing an older time window for your service that's beyond the retention period of your trace data. Select a time within the last week to see distributed tracing insights.
+* **No data from your entity** If your service is not reporting distributed tracing data. Follow links to our docs or the Guided Install to setup distributed tracing.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:104]:
```suggestion
## Understanding missing tracing data messages [#no-trace-data]
To ensure you're working with the most relevant distributed tracing insights, here's a breakdown of the messages you might encounter when data is unavailable, along with recommended actions:
* `No data for this time range`: 
     * This message indicates that you've selected a time window for your service that falls outside the current retention period for trace data.
     * To access insights, adjust the time range to focus on a period within the last week.
* `No data from your entity`:
     * This message suggests that your service isn't currently reporting distributed tracing data.
     * To unlock these valuable insights, set up distributed tracing for your service by following the [guided install](https://onenr.io/0OQM6lV3DwG). If you're new to distributed tracing, start by reading the [Distributed tracing: Planning guide](https://docs.newrelic.com/docs/distributed-tracing/concepts/distributed-tracing-planning-guide/).
* `No data from related entities`;
     * This message indicates your service is reporting data but its dependencies aren't.
     * By [enabling distributed tracing for your services](https://onenr.io/0OQM6lV3DwG), your dependencies should also be automatically enabled.
```

**Review by ghost — APPROVED** [review]

**Pull request merged**